mat64/format.go

```diff
@@ -11,13 +11,14 @@ import (
 
 // Format prints a pretty representation of m to the fs io.Writer. The format character c
 // specifies the numerical representation of of elements; valid values are those for float64
-// specified in the fmt package, with their associated flags. In addition to this, a '#' for
-// all valid verbs except 'v' indicates that zero values be represented by the dot character.
-// The '#' associated with the 'v' verb formats the matrix with Go syntax representation.
+// specified in the fmt package, with their associated flags. In addition to this, a space
+// preceding a verb indicates that zero values should be represented by the dot character.
 // The printed range of the matrix can be limited by specifying a positive value for margin;
```

Comment:
The margin comment is confusing. Is it trying to say that it just prints the boarder of the matrix?